.github/workflows/build.yml

```diff
@@ -46,6 +46,7 @@ jobs:
           -Dsonar.projectKey=${{ env.SONAR_PROJECT_KEY }}
           -Dsonar.organization=${{ env.SONAR_ORGANIZATION }}
           -Dsonar.host.url=https://sonarcloud.io
+          -Dsonar.coverage.jacoco.xmlReportPaths=$(find "$(pwd)" -path '*jacoco-aggregate/jacoco.xml')
```

Comment:
What about to specify exact path to "report-module" jacoco.xml?

Reply:
done